src/common/serviceRegistry.ts

```diff
@@ -0,0 +1,48 @@
+const defaultServiceRegistryInstance = {
+  taxonomyService: null,
+}
+
+export enum Services {
+  taxonomyService = 'taxonomyService',
+}
+
+const servicesByType = {
+  [Services.taxonomyService]: 'TaxonomyService',
+}
```

Comment:
maybe this can be the configuration of the registry and can be passed and overwrite